webapp-client/src/component/AddCategoryForm.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+export default class AddCategoryForm extends React.Component {
+  /** This function will be changed later to directly POST request to db instead of calling addCategory */
+  createCategory = (e) => {
+    e.preventDefault();
+    const category = this._inputElement.value;
+    this.props.addCategory(category);
+    this._inputElement.value = '';
+  };
+  render() {
+    return (
+      <form className="my-1" onSubmit={this.createCategory}>
+        <input
+          ref={(input) => (this._inputElement = input)}
+          type="text"
+          className="input is-rounded"
+          placeholder="New Category"
+        ></input>
+        <br></br>
```

Comment:
It would be better to use bulma to style the vertical spacing.

Reply:
We removed it:) and it looks fine without extra spacing